frontend/src/components/form/Password.vue

```diff
@@ -10,32 +10,42 @@ type Props = {
   name: string;
   placeholder?: string;
   type?: string;
+  bold?: boolean;
 };
 
 const props = withDefaults(defineProps<Props>(), {
   helptext: '',
   type: 'text',
   label: '',
-  placeholder: ''
+  placeholder: '',
+  bold: true
 });
 
 const { errorMessage, value } = useField<string>(toRef(props, 'name'));
 </script>
 
 <template>
   <div class="field col">
-    <label :for="props.name">{{ props.label }}</label>
+    <label
+      :class="{ 'font-bold': bold }"
+      :for="props.name"
+    >
+      {{ props.label }}
+    </label>
     <Password
       v-model.trim="value"
       :aria-describedby="`${props.name}-help`"
       :name="props.name"
       :type="props.type"
       :placeholder="props.placeholder"
+      class="w-full"
       :class="{ 'p-invalid': errorMessage }"
       :feedback="false"
       toggle-mask
     />
     <small :id="`${name}-help`">{{ helptext }}</small>
-    <ErrorMessage :name="props.name" />
+    <div>
+      <ErrorMessage :name="props.name" />
+    </div>
```

Comment:
Is the new div wrapper addressing some kind of strange spacing/layout issue? Usually we try to avoid nesting more DOM elements whenever possible.